src/prefect/tasks/notifications/email_task.py

```diff
@@ -90,25 +107,39 @@ def run(
                 provided at initialization
             - smtp_type (str, optional): either SSL or STARTTLS; defaults to the one provided
                 at initialization
+            - msg_plain (str, optional): the contents of the email, added as plain text can be used in
+                combination of msg; defaults to the one provided at initialization
+            - email_to_cc (str, optional): additional email address to send the message to as cc;
+                defaults to the one provided at initialization
+            - email_to_bcc (str, optional): additional email address to send the message to as bcc;
+                defaults to the one provided at initialization
 
         Returns:
             - None
         """
 
         username = cast(str, Secret("EMAIL_USERNAME").get())
         password = cast(str, Secret("EMAIL_PASSWORD").get())
-        email_to = cast(str, email_to)
 
-        contents = MIMEMultipart("alternative")
-        contents.attach(MIMEText(cast(str, msg), "html"))
+        message = EmailMessage()
+        message["Subject"] = subject
+        message["From"] = email_from
+        message["To"] = email_to
+        if email_to_cc:
+            message["Cc"] = email_to_cc
+        if email_to_bcc:
+            message["Bcc"] = email_to_bcc
 
-        contents["Subject"] = Header(subject, "UTF-8")
-        contents["From"] = email_from
-        contents["To"] = email_to
+        # https://docs.python.org/3/library/email.examples.html
+        # Add the html version. This converts the message into a multipart/alternative
+        # container, with the original text message as the first part and the new html
+        # message as the second part.
+        if msg_plain:
+            message.set_content(msg_plain, subtype="plain")
+        if msg:
+            message.add_alternative(msg, subtype="html")
```

Comment:
Sweet :D

Reply:
Starting with "Add the html version" then setting the plain message could be a little confusing

Reply:
Hm, yea agreed, kinda confusing now. I just copied that text, the order does actually matter. You cant first use add_alternative followed by set_content. 

Ill update it tomorrow!